src/interfaces/ffi/api.rs

```diff
@@ -147,6 +150,7 @@ pub unsafe extern "C" fn h_search(
     keywords_ptr: *const c_char,
     max_results_per_keyword: c_int,
     max_depth: c_int,
+    fetch_chains_batch_size: c_int,
```

Comment:
can we use a `c_uint` here ? and use `0` to trigger the default in the test below.

Reply:
I changed it but same as in JS… For example Flutter doesn't have unsigned integer so the user can send a `-1`